pkgs/development/tools/rust/cargo-public-api/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchCrate
+, pkg-config
+, openssl
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-public-api";
+  version = "0.12.2";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-ZpzR6A9mV6ARz2+SedVtLjNANOEj8Ks09AWcQooltug=";
+  };
+
+  cargoSha256 = "sha256-8BZ1fPzRSJysyLCmoMxk+8xOfmfIs7viHFCeSfnxvt8=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  # Tests fail
+  doCheck = false;
+
+  meta = with lib; {
+    description = "List and diff the public API of Rust library crates between releases and commits. Detect breaking API changes and semver violations";
+    homepage = "https://github.com/elic/cargo-public-api";
```

Comment:
```suggestion
    homepage = "https://github.com/Enselic/cargo-public-api";
```

Reply:
Thanks for catching that.

I guess this happened by fat-fingering something and because neovim currently has issues with visual artifacts, I missed the error. :cry: